readability-research.md

```diff
@@ -0,0 +1,127 @@
+- "traditional readability formulas rely on proxy estimates for lexical and syntactic features. For example, number of characters per word is used as a proxy for word sophistication, and number of words per sentence is used as a proxy for syntactic complexity"
+- "traditional formulas often ignore concerns of style, vocabulary, and grammar, which may equally play a role in text readability"
+- "the reading criteria used to develop traditional formulas are often based on multiple-choice questions and cloze tests, two methods that may not measure text comprehension accurately"
+- "most traditional readability formulas [...] have been normed using readers from specific age groups and using small corpora of texts from specific domains."
+	- The Flesch Reading Ease formula (Flesch, [1948](https://link.springer.com/article/10.3758/s13428-022-01802-x#ref-CR30 "Flesch, R. (1948). A new readability yardstick. Journal of Applied Psychology, 32(3), 221–233.")) and the Dale–Chall formula (Dale & Chall, [1948](https://link.springer.com/article/10.3758/s13428-022-01802-x#ref-CR25 "Dale, E., & Chall, J. S. (1948). A formula for predicting readability: Instructions. Educational Research Bulletin, 37–54.")) are both based on 350 texts from the 1920s for 3rd through 12th graders
+	- The Flesch Kincaid Grade Level formula was based on a small corpus from Navy training materials
+
+### CLEAR
+- Collaboration between CommonLit, a non-profit education technology organization focused on improving reading, writing, communication, and problem-solving skills, and Georgia State University (GSU)
+- Its objective is to promote the development of more advanced and open-source readability formulas, particularly for use by government agencies in testing, materials selection and creation, and more. 
+- CLEAR provides unique readability scores for ~ 5000 text excerpts along with information about the excerpt’s year of publishing, genre, and other metadata.
+
+#### Advantages of CLEAR
+- Large - Much larger than any available corpora that provide readability criterion based on human judgement. Other large corpora (such as Newsela) do not provide readability criterion for individual texts, and corpora that do are much smaller (N = ~60-200 texts)
+- Broad - Curated from texts available through CommonLit, which selected excerpts from Wikipedia, Project Gutenberg, and other digital libraries. The text excerpts were also published over a very long period of time (1791 - 2020). 
+- Texts were hand-selected and hand-edited, ensuring high quality and minimal errors. 
+- The CLEAR corpus also includes metadata which provides opportunity for subanalyses. 
+
+#### Limitations of CLEAR
+- Readability criteria are based on judgements from schoolteachers about how difficult the excerpts would be for their students to read. 
+
+
+## Paper 2: # Perusal of readability with focus on web content understandability
```

Comment:
I’ve not found this paper’s findings to be massively useful, though this [recap table](https://gist.github.com/thibaudcolas/d4b12d47d4ecffed98099655f3652a96) taken from it is very neat. From this, I believe there are only 6 different formulas which would be suitable for us to have built-in:

- ARI
- CLI
- LI×
- Raygor Estimate Graph
- Djoko Formula
- Lexile